demos/seasonal_contrast/README.md

```diff
@@ -0,0 +1,8 @@
+## Integrating the Seasonal Contrast project into the AI-platform.
+
+This project is focused on atmospheric transformation used to generate views for contrastive self-supervised learning on multi-spectralimages. Here, the experiments are benchmarked on the SeCo algorithm. By using atmospheric transformation as an alternative to color jittering, all the experiments conducted by authors of SeCo can be extended to multiple channels instead of RGB. The pre-trained models trained using atmospheric transformation shows higher accuracy over the two downstream tasks i.e. EuroSAT and Bigearthnet when compared with one trained with color jittering and grayscaling. The experiments run on both Juwels Booster and Juwels Cluster. Through the AI-platform it is possible to manage the pretraining and the training workflows and also to track the experiments and their results.<br/>
```

Comment:
```suggestion
This project is focused on atmospheric transformation used to generate views for contrastive self-supervised learning on multi-spectral images. Here, the experiments are benchmarked on the SeCo algorithm. By using atmospheric transformation as an alternative to color jittering, all the experiments conducted by authors of SeCo can be extended to multiple channels instead of RGB. The pre-trained models trained using atmospheric transformation show higher accuracy over the two downstream tasks i.e. EuroSAT and Bigearthnet when compared with one trained with color jittering and greyscaling. The experiments run on both JUWELS Booster and JUWELS Cluster. Through the AI-platform it is possible to manage the pretraining and the training workflows and also to track the experiments and their results.<br/>
```